### DISCUSSION TITLE
How to display images in jupyter with pyraf?

### DISCUSSION THREAD

**Discussion by daniellekelley19** [iraf-community/iraf#329] [Q&A]
Hello everyone! I'd like to use pyraf in jupyter notebook. It seems each pyraf command works okay in this environment except for those that can display images. 

Currently, I use "import pyraf" to initialise it and it returns me warning messages:
```
Unable to open CL script cache /root/.iraf/pyraf/clcache
Unable to open CL script cache /root/.iraf/pyraf/clcache.v2
Warning: unable to open any CL script cache, performance may be slow
Warning: sscanf library not installed on linux
```
Later when I invoke normal plot command as "implot(image="ha2d.fits001")", it just shows some errors:
```
Killing IRAF task 'implot'
PANIC in `/usr/lib/iraf/bin/x_plot.e': fatal error during process shutdown
IrafError: Unable to plot graphics to screen
```
Could anyone tell me how I can correctly displays the intermediate image files in jupyter with the pyraf package? Thanks in advance!

**Comment by olebole** [comment]
You wrote in #331
> Many tasks, like imslice, display and implot, could be soomthly invoked from the pyraf.iraf package just as illustrated in the tutorial. 

So, did you get it working? If yes, it would be great if you could describe what the problem was and how you solved it.
